deploy/helm/kubecf/assets/operations/instance_groups/app-autoscaler.yaml

```diff
@@ -995,7 +995,20 @@
     pre_render_scripts:
       jobs:
       - |
-        sed -i "s/sysctl/#sysctl/g" /var/vcap/all-releases/jobs-src/postgres/postgres/templates/pre-start.sh.erb
+        #!/usr/bin/env bash
+
+        set -o errexit -o nounset
+
+        target="/var/vcap/all-releases/jobs-src/postgres/postgres/templates/pre-start.sh.erb"
+        sentinel="${target}.patch_sentinel"
+        if [[ -f "${sentinel}" ]]; then
+          echo "Patch already applied. Skipping"
+          exit 0
+        fi
+
+        sed -i "s/sysctl/#sysctl/g" "${target}"
+
+        touch "${sentinel}"
```

Comment:
If the script needs to be this complicated, it should live in a separate file.

(Or we could just do `sed -i 's/^sysctl/#sysctl/g'` because that has no issues with re-patching)

Reply:
Done in https://github.com/SUSE/kubecf/pull/261/commits/059445e69f2611d0af7e51a29e5aa38ae9a45874.